test/Hydration-test.js

```diff
@@ -0,0 +1,111 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const Hydration = require('../src/Hydration.js');
+
+describe('Hydration', function() {
+  let hydrationData;
+
+  beforeEach(function() {
+    hydrationData = [
+      {
+        "userID": 1,
+        "date": "2019/06/15",
+        "numOunces": 37
+      },
+      {
+        "userID": 2,
+        "date": "2019/06/15",
+        "numOunces": 75
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/16",
+        "numOunces": 69
+      },
+      {
+        "userID": 2,
+        "date": "2019/06/16",
+        "numOunces": 91
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/17",
+        "numOunces": 96
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/18",
+        "numOunces": 100
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/19",
+        "numOunces": 20
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/20",
+        "numOunces": 31
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/21",
+        "numOunces": 91
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/22",
+        "numOunces": 55
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/23",
+        "numOunces": 74
```

Comment:
Using so much dummy data because we need (a) at least two user's data to ensure that user constructor filters data for just one user and (b) more than seven days of data to ensure that calculation of week's hydration returns only 7 days of data and the correct 7 days

Reply:
I appreciate the heads up on this one. I think it's fine. We might be able to find a way to dry it up later, but for now it works.